nntrainer/layers/loss/loss_layer.h

```diff
@@ -40,7 +40,13 @@ class LossLayer : public Layer {
   /**
    * @copydoc Layer::setProperty(const std::vector<std::string> &values)
    */
-  virtual void setProperty(const std::vector<std::string> &values) override {}
+  virtual void setProperty(const std::vector<std::string> &values) override {
+    if (!values.empty()) {
```

Comment:
I think below macro can be applied for convenience.
```cpp
NNTR_THROW_IF(!values.empty(), std::invalid_argument) 
        << "[Layer] Unknown Layer Properties count = " << values.size()
```

Reply:
done.